src/com/dre/brewery/utility/BScheduler.java

```diff
@@ -86,26 +86,26 @@ private static <T extends Throwable> T forciblyThrow(Throwable t) throws T {
 
 		@Override
 		public BTask runTaskAt(Location location, Consumer<BTask> task, long delay) {
-			var ret = p.getServer().getScheduler().runTaskLater(p, new Runner(task), delay);
+			var ret = new Runner(task).runTaskLater(p, delay);
```

Comment:
is this needed for the fix?

Reply:
likely yes, since the method is deprecated